src/TitleDetails/AddPieceModal/AddPieceModal.js

```diff
@@ -17,6 +17,7 @@ import {
   TextField,
   checkScope,
 } from '@folio/stripes/components';
+import { useOkapiKy } from '@folio/stripes/core';
 import stripesFinalForm from '@folio/stripes/final-form';
 import {
```

Comment:
add extra empty line please

Reply:
why?

Reply:
@folio and local imports will be identified easily.
![Screenshot 2023-10-12 at 2 16 41 PM](https://github.com/folio-org/ui-receiving/assets/116072773/2299bad7-acd5-44c5-993a-a6d26db7a14f)

Reply:
there is already an empty line between folio and local imports, you're showing a collapsed list :)